sheets/api.py

```diff
@@ -801,19 +813,19 @@ def get_sheet_rows(self, sheet_id):
         status_dates = (item_at_index_or_none(row, 3) for row in data_rows)
         return zip(coupon_codes, emails, statuses, status_dates)
 
-    def get_enumerated_sheet_rows(self, sheet_id):
+    def get_enumerated_sheet_rows(self, worksheet):
         """
         Returns enumerated raw data rows in a coupon assignment sheet.
 
         Args:
-            sheet_id (str): A coupon assignment spreadsheet id
+            worksheet (pygsheets.worksheet.Worksheet): A coupon assignment worksheet
 
         Returns:
             iterable of (int, iterable of (str, str, str, str)):
                 An enumerated matrix of raw row data from the sheet
         """
-        sheet_rows = self.get_sheet_rows(sheet_id)
-        return enumerate(sheet_rows, start=1)
+        sheet_rows = self.get_sheet_rows(worksheet)
+        return enumerate(sheet_rows, start=GOOGLE_SHEET_FIRST_ROW + 1)
```

Comment:
What does this change do? Are we skipping the first row because it's a header row?

Reply:
Correct